components/drawer/Drawer.razor.cs

```diff
@@ -517,6 +511,12 @@ private void CalcDrawerStyle()
         protected override void Dispose(bool disposing)
         {
             _timer?.Dispose();
+
+            if (_isOpen)
```

Comment:
@zxyao145 I changed here. Because if there are more than one drawer on one page, each their Dispose() would be called, and EnableBodyScroll would be invoke more then one times.

However, one problem is that if several drawers are opened at the same page, you need to recognize that all drawers are currently closed before calling EnableBodyScroll.